### PR TITLE
Update initial workerPool config value

### PR DESCRIPTION
**Pull request by Jo-IE**
Bugzilla Bug: [XXXXX](https://bugzilla.mozilla.org/show_bug.cgi?id=XXXXX)

### CODE REVIEW

**Comment by helfi92** [comment]
Adding @imbstack to the review since they suggested the idea.

File: ui/src/utils/constants.js [hunk 1]
```diff
@@ -204,14 +204,114 @@ export const PROVIDER_DEFAULT_CONFIGS = new Map([
...
+          scheduling: {
+            onHostMaintenance: 'terminate',
+          },
+          workerConfig: {
+            shutdown: {
+              enabled: true,
+            },
+          },
+        },
+      ],
```
Comment (imbstack) [inline ui/src/utils/constants.js]: Please remove these

File: ui/src/utils/constants.js [hunk 2]
```diff
@@ -204,14 +204,114 @@ export const PROVIDER_DEFAULT_CONFIGS = new Map([
...
+          zone: 'us-west1-a',
+          capacityPerInstance: 1,
+          disks: [
+            {
+              autoDelete: true,
+              boot: true,
+              initializeParams: {
+                diskSizeGb: 50,
+                sourceImage:
+                  'projects/taskcluster-imaging/global/images/docker-worker-gcp-googlecompute-2019-09-25t19-53-35z',
```
Comment (imbstack) [inline ui/src/utils/constants.js]: we can replace this with `...`

Comment (Jo-IE) [reply ui/src/utils/constants.js]: Hi @imbstack will I be replacing just the object or the `initializeParams` key as well?

Thanks in advance!

**Comment by helfi92** [comment]
@all-contributors please add @Jo-IE  for code.

**Comment by Jo-IE** [comment]
> LGTM, just missing a silent changelog to make the tests pass.

Done :)

**Pull request merged**